kong/tools/wrpc/proto.lua

```diff
@@ -0,0 +1,174 @@
+local pb = require "pb"
+local grpc = require "kong.tools.grpc"
+
+local grpc_new = grpc.new
+local pb_encode = pb.encode
+local setmetatable = setmetatable
+local string_format = string.format
+
+local _M = {}
+local _MT = { __index = _M, }
+
+local wrpc_proto_name = "wrpc.wrpc"
```

Comment:
I think it is a constant, so it should be `UPPERCASE`.

Reply:
What about `default_proto_path`?

Reply:
I am not sure about it ,maybe `WRPC_PROTO_NAME`